src/queries/hooks/channels/useVerifyAliasChain.ts

```diff
@@ -0,0 +1,13 @@
+import { useMutation } from '@tanstack/react-query';
+
+//Constants
```

Comment:
No need of import comments in any file. Please remove them from these changed files.